### PR TITLE
[DESING UPDATE] increase the font size of H1

### PR DESCRIPTION
**Pull request by corwintines**
**Changes**
- Remove redundant header1 style and update where it was used'
- Update fontSize for h1 in textStyles

Fixes: https://github.com/ethereum/geth-website/issues/114

### CODE REVIEW

File: src/components/UI/docs/MDComponents.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ import { Code, Note } from '.';
 import { textStyles } from '../../../theme/foundations';
 import { parseHeadingId } from '../../../utils/parseHeadingId';
 
-const { header1, header2, header3, header4 } = textStyles;
+const { h1, header2, header3, header4 } = textStyles;
```
Comment (nhsz) [inline src/components/UI/docs/MDComponents.tsx:18]: why do we have `header2`, `header3`, etc instead of `h2`, `h3`, etc?

**Review by nhsz — CHANGES_REQUESTED** [review]
Good, asking for a change

File: src/components/UI/docs/MDComponents.tsx (hunk 1 above)
Comment (corwintines) [reply src/components/UI/docs/MDComponents.tsx:18]: Will see what I can bring over to simplify

File: src/theme/foundations/textStyles.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ export const textStyles = {
   h1: {
     fontFamily: 'heading',
     fontWeight: 700,
-    fontSize: '2.75rem',
+    fontSize: { base: '2.175rem', md: '2.875rem' },
```
Comment (wackerow) [inline src/theme/foundations/textStyles.ts:5]: I'm confused here... 
- the heading1 being removed below uses 2.125rem desktop/1.875 mobile
- this is proposing 2.875 desktop/2.175 mobile
- and the design system shows 2.175rem for both desktop and mobile

@nloureiro Could you help clarify here what the fontSize should be for the standard H1?

Comment (wackerow) [reply src/theme/foundations/textStyles.ts:5]: Ah, never mind, I see #114 now. @nloureiro Would you mind updating the design system to reflect this?

File: src/theme/foundations/textStyles.ts [hunk 3]
```diff
@@ -15,43 +15,28 @@ export const textStyles = {
...
-    fontFamily: 'heading',
-    fontWeight: 700,
-    fontSize: { base: '1.875rem', md: '2.125rem' },
-    letterSpacing: { base: '0.03rem', md: '0.04rem' },
-    lineHeight: 'normal',
-    color: 'body'
-  },
-  header2: {
-    fontFamily: 'heading',
-    fontSize: { base: '1.5rem', md: '1.75rem' },
```
Comment (nloureiro) [inline src/theme/foundations/textStyles.ts:28]: it does not have a fontWeight defined: after checking that a Figma bug shows in the inspectors 400 it really should be 700.

**Review by nhsz — CHANGES_REQUESTED** [review]
update: `h2` should have fontWeight: 700, the rest is ok!

**Comment by corwintines** [comment]
Failing build from rate limiting, going to merge based on previous approval and fixed fontWeight.

**Comment by wackerow** [comment]
Yeah, agree, this builds fine for me locally

**Pull request merged**